src/components/EmojiPicker/EmojiPickerMenuItem/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useEffect, useRef, useState} from 'react';
+import PressableWithoutFeedback from '@components/Pressable/PressableWithoutFeedback';
+import Text from '@components/Text';
+import useStyleUtils from '@hooks/useStyleUtils';
+import useThemeStyles from '@hooks/useThemeStyles';
+import * as Browser from '@libs/Browser';
+import getButtonState from '@libs/getButtonState';
+import CONST from '@src/CONST';
+import type EmojiPickerMenuItemProps from './types';
+
+function EmojiPickerMenuItem({emoji, onPress, onHoverIn, onHoverOut, onFocus, onBlur, isFocused, isHighlighted}: EmojiPickerMenuItemProps) {
+    const [isHovered, setIsHovered] = useState(false);
+    const ref = useRef<HTMLDivElement | null>(null);
+    const StyleUtils = useStyleUtils();
+    const themeStyles = useThemeStyles();
+
+    const focusAndScroll = () => {
+        ref?.current?.focus({preventScroll: true});
+        ref?.current?.scrollIntoView({block: 'nearest'});
+    };
+
+    useEffect(() => {
+        if (!isFocused) {
+            return;
+        }
+        focusAndScroll();
+    }, [isFocused]);
+
+    return (
+        <PressableWithoutFeedback
+            shouldUseAutoHitSlop={false}
+            onPress={() => onPress(emoji)}
```

Comment:
If emoji from props is required, most likely the functions like onPress should also make emoji as a required argument